tests/deepspeed/test_deepspeed.py

```diff
@@ -218,14 +238,15 @@ def setUp(self):
         with io.open(self.ds_config_file[ZERO2], "r", encoding="utf-8") as f:
             config_zero2 = json.load(f)
             # by default use fp16
-            config_zero2["fp16"]["enabled"] = True
+            # config_zero2["fp16"]["enabled"] = True
```

Comment:
Why leave it commented out? I understand the print statements are for quick debugging, but this should be cleaned up, no?

Reply:
thank you for catching - cleaned up!